packages/lib/src/artifacts/ZWeb3.ts

```diff
@@ -18,6 +18,11 @@ const NETWORKS = {
   42: 'kovan'
 };
 
+export interface TxParams {
```

Comment:
Could you double check whether `txParams` can include gasPrice as well? Clearly we are not using them from our code (or the compiler would have complained), but it would be useful to know if we are passing gasPrice onto web3 at any point.